.github/workflows/conformance-gateway-api.yaml

```diff
@@ -131,7 +131,7 @@ jobs:
         run: |
           echo sha=${{ steps.default_vars.outputs.sha }} >> $GITHUB_OUTPUT
           
-          EXEMPT_FEATURES="GatewayStaticAddresses,HTTPRouteParentRefPort,HTTPRouteResponseHeaderModification"
```

Comment:
Sadly, this change will not be reflected in CI on forked repo, so you might need to run the test locally and share the result here, or push your  change to cilium/cilium branch,  and  create another branch.

https://github.com/cilium/cilium/pull/33042#issuecomment-2160814887